packages/api-client/api-types/storeApiTypes.d.ts

```diff
@@ -302,41 +286,41 @@ export type Schemas = {
   };
   ArrayStruct: components["schemas"]["Struct"];
   Association: {
-    [key: string]: components["schemas"]["Association"];
+    [key: string]: components["schemas"]["Association"] | undefined;
```

Comment:
This is the main change that still should be resolved. We don't want to `undefined` values here if the schema correctly suggest types which are here